### PR TITLE
task2, ver.1

### PR DESCRIPTION
**Pull request by tatianaklimenko**
Review please.

### CODE REVIEW

File: Task2 - Delegates/tatiana-klimenko/Delegates/Delegates/Program.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Delegates
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```
Comment (chesdenis) [inline Task2 - Delegates/tatiana-klimenko/Delegates/Delegates/Program.cs]: empty fragment

Comment (tatianaklimenko) [reply Task2 - Delegates/tatiana-klimenko/Delegates/Delegates/Program.cs]: In solution explorer i can't see "delegates" project, but in file manager there's one. Can I delete the whole folder? Why does it happens?

Comment (chesdenis) [reply Task2 - Delegates/tatiana-klimenko/Delegates/Delegates/Program.cs]: This is beacause git does not know anything about vs solution. It works just with file system. So please remove whole folder.

Comment (tatianaklimenko) [reply Task2 - Delegates/tatiana-klimenko/Delegates/Delegates/Program.cs]: ok

**Pull request merged**